packages/graphql/src/classes/Node.ts

```diff
@@ -53,6 +54,7 @@ export interface NodeConstructor {
     ignoredFields: BaseField[];
     auth?: Auth;
     exclude?: Exclude;
+    nodeDirective?: NodeDirective;
```

Comment:
naming conventions are a bit inconsistent here we may want to discuss if is worth it adding ...Directive to all directives